pyproject.toml

```diff
@@ -76,6 +76,10 @@ doc.shell = "cd docs && mkdocs serve -a localhost:8030"
 lint.help = "Run linters over all files"
 lint.cmd = "pre-commit run --all-files"
 
+# PDM hook to automatically sort dependencies when they change.
+# "true" is there since GitHub Actions will fail if this returns a non-zero exit code.
+post_install.shell = "pre-commit run keep-sorted --files pyproject.toml > /dev/null 2> /dev/null; true"
```

Comment:
The redirects can shorten to `2>&1 > /dev/null`

I'm not sure why you want to hide stderr and ignore failures, though. Is there an audience that needs to be able to make precommit errors and not know about them? Should there be a "soft" precommit next to a "full" one?